tests/e2e/e2e_test.go

```diff
@@ -200,7 +211,7 @@ var _ = ginkgo.Describe("[Start/Remove/Restart/Add/Stop]", func() {
 
 	ginkgo.It("can wait for health", func() {
 		// start is async, so wait some time for cluster health
-		time.Sleep(2 * time.Minute)
+		time.Sleep(30 * time.Second)
```

Comment:
Can we just poll every 10 seconds for health instead of sleeping here?

Reply:
Can we please do this in a separate PR? This pattern is used all over the e2e tests and it has not been introduced with this PR.

Reply:
Agreed. I will also create a ticket right away.